pages/visitors/[id]/start.js

```diff
@@ -49,4 +54,17 @@ const Start = () => {
   );
 };
 
+export const getServerSideProps = propsWithContainer(
```

Comment:
Should probably add some tests for this :eyes:

Reply:
Added more tests :D